### ISSUE TITLE
Batch Size Limit for Large Transaction

### ISSUE THREAD

**Issue by handysama** [kfsoftware/hlf-operator#96]
Recently, i encounter an error when try to put transaction more than 1 Mb.

```bash
# Reformatted for clarity
2022/05/25 12:10:25 Error in PutTemplate: 
    CreateAndSendTransaction failed: 
    SendTransaction failed: calling orderer '172.31.3.234:31474' failed: 
    Orderer Server Status Code: (400) BAD_REQUEST. 
    Description: message payload is 1867595 bytes and exceeds maximum allowed 1048576 bytes
```

I traced the limit is set by this line of code of `testutils.GetApplicationChannelBlock`.

https://github.com/hyperledger-labs/hlf-operator/blob/94c333140de92a1125d9fba8192396a01afbed4b/controllers/testutils/channel.go#L183

https://github.com/hyperledger-labs/hlf-operator/blob/94c333140de92a1125d9fba8192396a01afbed4b/kubectl-hlf/cmd/channel/generate.go#L51

https://github.com/hyperledger-labs/hlf-operator/blob/94c333140de92a1125d9fba8192396a01afbed4b/kubectl-hlf/cmd/channel/generate.go#L177

To solved this issue, i bump `AbsoluteMaxBytes` to 10 Mb (10 * 10124 * 1024) and recompile `kubectl-hlf`, replace existing one, and recreate the cluster. After that, i can write transaction > 1 Mb.

My question is: can we add parameter to set the batch size limit? Or, is there any proper approach to override the limit?

**Comment by dviejokfs** [comment]
Moved to milestone to 1.7.0

**Comment by dviejokfs** [comment]
Implemented in 1.7.0
